analyzers/PapasSim.py

```diff
@@ -103,55 +95,40 @@ def init_display(self):
 
     def process(self, event):
         '''
-           event must contain the
-           
+           event must contain
+             todo once history is implemented
```

Comment:
you might want to put this documentation in the main docstring of this analyzer class, as done for the other analyzers .